frontend/src/metabase/querying/utils/drills/column-extract-drill/column-extract-drill.tsx

```diff
@@ -38,3 +39,35 @@ export const columnExtractDrill: Drill<Lib.ColumnExtractDrillThruInfo> = ({
     },
   ];
 };
+
+export function getExample({ key }: Lib.ColumnExtraction) {
+  if (key === "hour-of-day") {
```

Comment:
Would it be better to add the examples in MLv2, so we don't need to change anything on the FE when new extractions get added?

@bshepherdson what do you think?